core/kazoo_amqp/src/api/kz_api.erl

```diff
@@ -285,7 +285,6 @@ do_empty_value_removal([{K,V}=KV|T], Recursive, Acc) ->
 
 -spec is_empty(any()) -> boolean().
 is_empty('undefined') -> 'true';
-is_empty([]) -> 'true';
```

Comment:
@jamesaimonetti this could be significant, I would like some more time to look into the ramifications (or further discussion during tomorrows standup).